install_config/persistent_storage/topics/glusterfs_example_full.adoc

```diff
@@ -0,0 +1,122 @@
+
+openshift_storage_glusterfs_block_deploy=false
+
+openshift_storage_glusterfs_registry_block_deploy=true
+openshift_storage_glusterfs_registry_block_storageclass=true
+openshift_storage_glusterfs_registry_block_storageclass_default=true
+
+openshift_storageclass_default=false
+----
+<1> It is recommended to run the integrated OpenShift Container Registry on
+nodes dedicated to "infrastructure" applications, that is applications deployed
+by administrators to provide services for the {product-title} cluster.
+
+. Add `glusterfs` and `glusterfs_registry` in the `[OSEv3:children]` section to
+enable the `[glusterfs]` and `[glusterfs_registry]` groups:
++
+----
+[OSEv3:children]
+...
+glusterfs
+glusterfs_registry
+----
+
+. Add `[glusterfs]` and `[glusterfs_registry]` sections with entries for each
+storage node that will host the GlusterFS storage. For each node, set
+`glusterfs_devices` to a list of raw block devices that will be completely
+managed as part of a GlusterFS cluster. There must be at least one device
+listed. Each device must be bare, with no partitions or LVM PVs. Specifying the
+variable takes the form:
++
+----
+<hostname_or_ip> glusterfs_devices='[ "</path/to/device1/>", "</path/to/device2>", ... ]'
+----
++
+For example:
++
+----
+[glusterfs]
+node11.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+node12.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+node13.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+
+[glusterfs_registry]
+node14.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+node15.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+node16.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+----
+
+. Add the hosts listed under `[glusterfs]` and `[glusterfs_registry]` to the
+`[nodes]` group:
++
+----
+[nodes]
+...
+node11.example.com openshift_schedulable=True openshift_node_labels="{'role': 'app'}"   <1>
+node12.example.com openshift_schedulable=True openshift_node_labels="{'role': 'app'}"   <1>
+node13.example.com openshift_schedulable=True openshift_node_labels="{'role': 'app'}"   <1>
+node14.example.com openshift_schedulable=True openshift_node_labels="{'role': 'infra'}" <1>
+node15.example.com openshift_schedulable=True openshift_node_labels="{'role': 'infra'}" <1>
+node16.example.com openshift_schedulable=True openshift_node_labels="{'role': 'infra'}" <1>
+----
+<1> The nodes are marked to denote whether they will allow general applications
+or infrastructure applications to be scheduled on them. It is up to the
+administrator to configure how applications will be constrained.
+
+. Run the Advanced Installer without deploying Logging or Metrics. This can be
+either greenfield:
++
+----
+ansible-playbook -i <path_to_inventory_file> /usr/share/ansible/openshift-ansible/playbooks/prerequisites.yml
+
+ansible-playbook -i <path_to_inventory_file> /usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.yml
+----
++
+or brownfield:
++
```

Comment:
We should explain the terms greenfield/brownfield (while these are industry terms, they are a form of jargon).

Reply:
...oh geez, yeah. Actually, I kinda don't like them in general (I think I was just rushed) so I'll find a better wording. :)